README.adoc

```diff
@@ -198,31 +198,31 @@ include::finish/src/main/java/io/openliberty/guides/testing/PersonService.java[]
 
 == Writing your first test
 
-Now that the setup is complete, we can write our first test case. We will start by testing the basic "create person" use case for our REST-based application. To do this, we use the REST client injected by MicroShed Testing to make the HTTP POST request to our application and read the response:
+Now that the setup is complete, you can write your first test case. Start by testing the basic `create person` use case for your REST-based application. To test this use case, use the REST client that's injected by MicroShed Testing to make the HTTP POST request to the application and read the response.
```

Comment:
I think "create person" should be in quotes, rather than a `code block` because it's a term we are referring to rather than a  piece of code